internal/version.go

```diff
@@ -43,7 +43,7 @@ package internal
 // Due to all of this unreliability, this should be used as strictly informational
 // metadata, e.g. for caller version monitoring, never behavioral (use
 // FeatureVersion or feature flags instead).
-const LibraryVersion = "1.1.0"
+const LibraryVersion = "1.2.6"
```

Comment:
Why is it 1.1.0 -> 1.2.6? Is it to align with something?

Reply:
Due to pushing invalid versions, the latest version is still 1.2.5, but it's incorrect and we need to release a greater version to update the version in proxy.golang.org, so the next one is 1.2.6.

Reply:
got it!